src/Components/README.md

```diff
@@ -61,10 +73,23 @@ source activate.sh
 . ./activate.ps1
 ```
 
-4. Now you can build `src\Components\Web.JS\` by running the following commands in the `src\Components\Web.JS` directory:
+1. Now you can build all the JavaScript assets required by the repository (including SignalR for instance) by running the following command:
```

Comment:
It's probably implied, but it might be helpful to explicitly say that you should run this in the repo root.

Reply:
I made the commands "stateless" as you suggested. I think it doesn't need to be mentioned anymore as the previous command is also at the root.

Reply:
... and this command also works in a subfolder apparently (just tried)